rules/integrations/aws/defense_evasion_ec2_flow_log_deletion.toml

```
@@ -22,8 +22,62 @@ index = ["filebeat-*", "logs-aws*"]
 interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
-name = "AWS EC2 Flow Log Deletion"
-note = """## Setup
+name = "AWS VPC Flow Logs Deletion"
+note = """## Triage and analysis
+
+### Investigating AWS VPC Flow Logs Deletion
+
+VPC Flow Logs is a feature that enables you to capture information about the IP traffic going to and from network
+interfaces in your VPC. Flow log data can be published to Amazon CloudWatch Logs or Amazon S3.
```

Adding a little helpful context:
```suggestion
VPC Flow Logs is an AWS feature that enables you to capture information about the IP traffic going to and from network
interfaces in your virtual private cloud (VPC). Flow log data can be published to Amazon CloudWatch Logs or Amazon S3.
```